benchmark/scripts/compare_perf_tests.py

```diff
@@ -308,7 +308,7 @@ def _reset(self):
 
     # Parse lines like this
     # #,TEST,SAMPLES,MIN(μs),MAX(μs),MEAN(μs),SD(μs),MEDIAN(μs)
-    results_re = re.compile(r'(\d+[, \t]*\w+[, \t]*' +
+    results_re = re.compile(r'([ ]*\d+[, \t]*\w+[, \t]*' +
```

Comment:
nitpick: brackets are not required around the single space

Reply:
I did not know that space and a star would work. This way it is in same style as all the other repeated groups. Do you want me to change it?